GovUk.Frontend.Umbraco/uSync/v9/ContentTypes/govuknotificationbanner.config

```diff
@@ -21,16 +21,32 @@
   </Info>
   <Structure />
   <GenericProperties>
+    <GenericProperty>
+      <Key>0e4f018c-2e4c-4d49-8af1-1ab9b6a3e2d8</Key>
+      <Name>Blocks</Name>
+      <Alias>blocks</Alias>
+      <Definition>b02f5487-5e04-4422-9b8c-e1c9429e9e2e</Definition>
+      <Type>Umbraco.BlockList</Type>
+      <Mandatory>false</Mandatory>
+      <Validation></Validation>
+      <Description><![CDATA[]]></Description>
+      <SortOrder>2</SortOrder>
+      <Tab Alias="content">Content</Tab>
+      <Variations>Nothing</Variations>
+      <MandatoryMessage></MandatoryMessage>
+      <ValidationRegExpMessage></ValidationRegExpMessage>
+      <LabelOnTop>false</LabelOnTop>
+    </GenericProperty>
     <GenericProperty>
       <Key>c058f0d1-8ae3-4cac-97d9-b3b561f5e924</Key>
       <Name>Heading</Name>
       <Alias>heading</Alias>
       <Definition>f59f7848-da0b-4eb5-8f57-0b96bca1264b</Definition>
       <Type>Umbraco.TinyMCE</Type>
-      <Mandatory>true</Mandatory>
+      <Mandatory>false</Mandatory>
```

Comment:
This probably should still be mandatory, unless Jodi has told you otherwise. But if it isn't you need to update the view because it always renders an h3 at the moment even if there is no content.

Reply:
Jodi has said there could be scenarios where we don't want a heading but do want a body. I have updated the view so the `h3` isn't rendered if the heading isn't present